File: ListasAtiv.PYTHON/Questao7.py
# Escreva uma função que receba uma lista de strings e retorne uma nova lista com todas
# as strings em maiúsculas.

def transformaEmMaiusculo(texto):
    return texto.upper()

def transformaListaEmMaiusculo(lista):
    listaMaiuscula = [transformaEmMaiusculo(x) for x in lista]
    return listaMaiuscula

lista = ["teclado","disciplina","notebook"]
print(transformaListaEmMaiusculo(lista))

File: ListasAtiv.HASKELL/Questao7.hs
-- Escreva uma função que receba uma lista de strings e 
-- retorne uma nova lista com todas
-- as strings em maiúsculas

import Data.Char (toUpper)

transformaEmMaiusculo :: String -> String
transformaEmMaiusculo texto = 
    map toUpper texto

geraListaMaiuscula listaMinusculas = 
    [transformaEmMaiusculo x | x <-listaMinusculas]    

main = do
    let lista = ["teclado", "aula","notebook"]
    print(geraListaMaiuscula lista)

